build-webkit: Remove AppleWin and FTW ports support

fujii opened the pull request:
#### 0302892f54aa7b598e1660e26fe8a416f8bbb297
<pre>
build-webkit: Remove AppleWin and FTW ports support
<a href="https://bugs.webkit.org/show_bug.cgi?id=252882">https://bugs.webkit.org/show_bug.cgi?id=252882</a>

Reviewed by Don Olmstead.

* Tools/Scripts/build-webkit:
* Tools/Scripts/update-webkit-auxiliary-libs: Removed.
* Tools/Scripts/update-webkit-support-libs: Removed.
* Tools/Scripts/webkitdirs.pm:
(usesPerConfigurationBuildDirectory):
(determineConfigurationProductDir):
(builtDylibPathForName):
(determinePortName):
(isAppleWebKit):
(launcherName):
(setPathForRunningWebKitApp):
(runSafari):
(runMiniBrowser):
(isFTW): Deleted.
(isAppleWinWebKit): Deleted.
(checkInstalledTools): Deleted.
(setupAppleWinEnv): Deleted.
(appleApplicationSupportPath): Deleted.

Canonical link: <a href="https://commits.webkit.org/260811@main">https://commits.webkit.org/260811@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2685aeb89400ee936ae3b58dc479f50156c9f0e6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/109439 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/18563 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/42201 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/87/builds/937 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/118603 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/113324 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/20028 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9760 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/101714 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/115195 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14924 "Passed tests") | [❌ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/98145 "Found 1 new API test failure: TestWebKitAPI.VideoControlsManager.VideoControlsManagerMultipleVideosSwitchControlledVideoWhenScrolling (failure)") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/43120 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/96896 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29798 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/84878 "Found 2 new API test failures: /TestWebKit:WebKit.OnDeviceChangeCrash, /TestWebKit:WebKit.PrivateBrowsingPushStateNoHistoryCallback (failure)") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/1/builds/98357 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/11307 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/31141 "Passed tests") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/38/builds/99250 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/83/builds/9330 "Built successfully and passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11971 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/8076 "Passed tests") | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/46/builds/31062 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/17337 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50745 "Passed tests") | [✅ 🛠 jsc-mips](https://ews-build.webkit.org/#/builders/37/builds/107222 "Built successfully") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7488 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/13702 "Built successfully") | | [✅ 🧪 jsc-mips-tests](https://ews-build.webkit.org/#/builders/45/builds/26471 "Passed tests") | 
| | | | | 
<!--EWS-Status-Bubble-End-->

stwrt reviewed (changes requested):
Looks like you missed removing the `isFTW` call here.

https://github.com/WebKit/WebKit/blob/main/Tools/Scripts/webkitdirs.pm#L717

donny-dont reviewed (approved):
Do you want the script to be able to launch MiniBrowser at all? I can't say I've used it so I don't have a strong opinion.

fujii committed:
build-webkit: Remove AppleWin and FTW ports support
https://bugs.webkit.org/show_bug.cgi?id=252882

Reviewed by Don Olmstead.

* Tools/Scripts/build-webkit:
* Tools/Scripts/update-webkit-auxiliary-libs: Removed.
* Tools/Scripts/update-webkit-support-libs: Removed.
* Tools/Scripts/webkitdirs.pm:
(usesPerConfigurationBuildDirectory):
(determineConfigurationProductDir):
(builtDylibPathForName):
(determinePortName):
(isAppleWebKit):
(launcherName):
(setPathForRunningWebKitApp):
(runSafari):
(runMiniBrowser):
(isFTW): Deleted.
(isAppleWinWebKit): Deleted.
(checkInstalledTools): Deleted.
(setupAppleWinEnv): Deleted.
(appleApplicationSupportPath): Deleted.

Canonical link: https://commits.webkit.org/260811@main
